shell-ghc-9-0.nix

```diff
@@ -4,7 +4,10 @@ let
   pkgs = import sources.nixpkgs-unstable { };
 in import nix/mk-shell.nix {
   pkgs = pkgs;
-  haskellPackages = pkgs.haskell.packages.ghc901;
-  useServant_0_18_3 = true;
-  customSafeColouredText = false;
+  haskellPackages = pkgs.haskell.packages.ghc901.extend (self: super: {
```

Comment:
much nicer. I tried this approach initially but gave up "trying to fix too many problems at once"